sdk/mediaservices/ci.yml

```diff
@@ -15,7 +15,7 @@ pr:
   branches:
     include:
       - master
-    - main
+      - main
```

Comment:
This change seems unrelated to the PR?

Reply:
I agree they seem unrelated but perhaps you hit an issue with this pipeline while testing. In any case we need to fix this and make sure this problem isn't repeated in other ci.yml files cc @sima-zhu

Reply:
The change is actually a fix, and is intended else it will not be parsed by the yml parser we use.